packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -275,6 +275,8 @@ pay-update-card-exp = Expires { $expirationDate }
 pay-update-change-btn = Change
 
 ## reactivate
+# Variables:
+    # $date (FluentDateTime) - Last day of product access
```

Comment:
Comments starting with a single `#` are either standalone, or attached to the message immediately below. In this case, that's `reactivate-confirm-dialog-header` which doesn't use a `$date` variable (but it references `$name`).

You can either put a single-`#` comment above each message, or you can edit the group comment. Group comments start with two `##` and apply to all messages below them, until the next group comment. In Pontoon, localizers will see both the group and the message comment when they edit a translation.

See https://projectfluent.org/fluent/guide/comments.html for more info about comments in Fluent.